.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 100,
+  "singleQuote": true
```

Comment:
One slight difference from the main repo, this repo uses primarily single quotes.